app/src/main/java/org/mozilla/focus/telemetry/AdjustConstant.kt

```diff
@@ -0,0 +1,3 @@
+package org.mozilla.focus.telemetry
+
+const val EVENT_TAKE_SCREENSHOT = "ky29gk"
```

Comment:
Is this randomly generated id? do we need to sync this with @elin-moco ?

Reply:
@elin-moco give me this key. it's generated by Adjust backend. We can't change it.